e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -189,3 +192,31 @@ func InitializeIngestManagerTestSuite(ctx *godog.TestSuiteContext) {
 		}
 	})
 }
+
+var opts = godog.Options{
+	Output: colors.Colored(os.Stdout),
+	Format: "progress", // can define default values
+}
+
+func init() {
+	godog.BindCommandLineFlags("godog.", &opts) // godog v0.11.0 (latest)
+}
+
+func TestMain(m *testing.M) {
```

Comment:
Could you explain the rationale for adding this support? I guess it's for `go test` compatibility, right?

Besides that, should we add this `TestMain` method to the rest of test suites (k8s, metricbeat, helm)?

Reply:
Yea we need this b/c godog doesn't compile on windows